datastore_entity/datastore_entity.py

```diff
@@ -467,28 +467,23 @@ def get_objects(self, prop, value, limit=500, paginate=False, cursor=None):
         query_res = query.fetch(start_cursor=cursor, limit=limit)
 
         if paginate:
-            query_res = query.fetch(start_cursor=cursor, limit=limit)
             current_page = next(query_res.pages)
             entities = list(current_page)
             next_cursor = query_res.next_page_token.decode('utf-8')
         else:
-            entities = list(query.fetch(start_cursor=cursor, limit=limit))
+            entities = list(query_res)
 
-        if entities:
-            objs = []
-            for entity in entities:
-
-                # get all properties. note that properties may not be
-                # defined in the class
-                for k, v in entity.items():
-                    setattr(self, k, v)
+        objs = []
+        for entity in entities:
+            # get all properties. note that properties may not be
+            # defined in the class
+            for k, v in entity.items():
+                setattr(self, k, v)
 
-                self.key = entity.key
+            self.key = entity.key
 
-                objs.append(self)
-            return (objs, next_cursor)
-        else:
-            return None
```

Comment:
I didn't realise this would go in as well (bad PR etiquette on my part - sorry :s ). 

Just to draw your attention to this being a  breaking change in the case of `None`